checks/evaluation/pinned_dependencies_test.go

```diff
@@ -111,7 +111,7 @@ func Test_PinningDependencies(t *testing.T) {
 				Error:         nil,
 				Score:         checker.MaxResultScore,
 				NumberOfWarn:  0,
-				NumberOfInfo:  7,
+				NumberOfInfo:  8,
```

Comment:
similar fyi: this is also something that #3254 should probably fix.

Reply:
Ack!